docs/developer-guide/syntax-reference.rst

```diff
@@ -1620,7 +1620,9 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | RANDOM                 |  ``RANDOM()``                                                             | Return a random DOUBLE value between 0.0 and 1.0. |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
-| ROUND                  |  ``ROUND(col1)``                                                          | Round a value to the nearest BIGINT value.        |
+| ROUND                  |  ``ROUND(col1)`` or ``ROUND(col1, precision)``                            | Round a value to the nearest BIGINT value.        |
+|                        |                                                                           | precision is an INT indicating the number.        |
```

Comment:
```suggestion
|                        |                                                                           | scale is an INT indicating the number.        |
```

Reply:
I'd also document that a negative number will round to the left of the decimal point, e.g. 

`ROUND(111, -1)` results in `110`.

Reply:
https://github.com/confluentinc/ksql/pull/3028/commits/ad0b1dd2a17a76771986263b2c3f83a35d853e1c